tests/test_stamp.py

```diff
@@ -149,3 +148,141 @@ def test_stamp_nan_zero_mean_self_join():
     naive.replace_inf(ref_mp)
     naive.replace_inf(comp_mp)
     npt.assert_almost_equal(ref_mp[:, :2], comp_mp)
+
+
+def test_stamp_mass_PI_with_isconstant_case1():
+    # case1: The query `Q` is not constant
+    T_B = np.random.uniform(-1, 1, [64])
+    isconstant_custom_func = functools.partial(
+        naive.isconstant_func_stddev_threshold, stddev_threshold=0.5
+    )
+
+    m = 3
+    zone = int(np.ceil(m / 2))
+
+    T_B_subseq_isconstant = naive.rolling_isconstant(T_B, m, isconstant_custom_func)
+    M_T, Σ_T = core.compute_mean_std(T_B, m)
+
+    trivial_idx = np.random.choice(np.flatnonzero(~T_B_subseq_isconstant))
+    Q = T_B[trivial_idx : trivial_idx + m]
```

Comment:
In this test function, I am choosing a Q that is not constant. And, in the next test function, I am choosing a Q that is constant.

Reply:
Maybe add a note

Reply:
@seanlaw 
There is actually a comment just in the first line of this test function. 
```
def test_stamp_mass_PI_with_isconstant_case1():
    # case1: The query `Q` is not constant
```

Does that suffice?

(There is a similar comment for the test function "test_stamp_mass_PI_with_isconstant_case2")